package.json

```diff
@@ -298,19 +295,22 @@
     "prettier": "^2.6.2",
     "prop-types": "^15.8.1",
     "pug-loader": "^2.4.0",
+    "qs": "^6.10.5",
```

Comment:
Nit: I know technically webpack bundles all dependencies regardless of which section they are in but I believe "qs" is used for more than just development!